src/components/Dashboard/index.js

```diff
@@ -232,13 +235,18 @@ class Dashboard extends Component {
       market =>
         process.env.WHITELIST[market.creator] && !market.oracle.isOutcomeSet && !market.event.isWinningOutcomeSet,
     )
-    const newMarkets = whitelistedMarkets.filter(market => new Date() - new Date(market.creationDate) < oneDayHours)
+    const newMarkets = whitelistedMarkets
+      .filter(market => new Date() - new Date(market.creationDate) < oneDayHours)
+      .sort((a, b) => a.creationDate > b.creationDate)
```

Comment:
can you move this into an external constant? e.g. `const sortByCreationDate = (a, b) => a.creationDate > b.creationDate` at the top of the file? otherwise this will get recreated every render.